website/source/docs/providers/nomad/r/job.html.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: "nomad"
+page_title: "Nomad: nomad_job"
+sidebar_current: "docs-nomad-resource-job"
+description: |-
+  Manages a job registered in Nomad.
+---
+
+# nomad\_job
+
+Manages a job registered in Nomad.
+
+This can be used to initialize your cluster with system jobs, common services,
+and more. In day to day Nomad use it is common for developers to submit
+jobs to Nomad directly, but jobs that must always be running or system-level
+jobs can be registered with Terraform.
+
+## Example Usage
+
+```
+resource "nomad_job" "app" {
+    jobspec = "${file("${path.module}/job.hcl")}"
```

Comment:
I doubt anyone is ever going to write a whole jobspec inline but maybe an example of a jobspec somewhere so folks know that it is they are looking for?